shared/xccdf/system/software/integrity.xml

```diff
@@ -98,23 +98,153 @@ must be captured and it should be able to be verified against the installed file
+If AIDE has already been configured for periodic execution in <tt>/etc/crontab</tt>, append the
+following line to the existing AIDE line:
+<pre> | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+Otherwise, add the following line to <tt>/etc/crontab</tt>:
+<pre>05 4 * * * root /usr/sbin/aide --check | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+AIDE can be executed periodically through other means; this is merely one example.
+</description>
+<ocil clause="AIDE has not been configured or has not been configured to notify personnel of scan details">
+To determine that periodic AIDE execution has been scheduled, run the following command:
+<pre>$ grep aide /etc/crontab</pre>
+The output should return something similar to the following:
+<pre>05 4 * * * root /usr/sbin/aide --check | /bin/mail -s "$(hostname) - AIDE Integrity Check" root@localhost</pre>
+</ocil>
+<rationale>
+Unauthorized changes to the baseline configuration could make the system vulnerable
+to various attacks or allow unauthorized access to the operating system. Changes to
+operating system configurations can have unintended side effects, some of which may
+be relevant to security.
+<br/><br/>
+Detecting such changes and providing an automated response can help avoid unintended,
+negative consequences that could ultimately affect the security state of the operating
+system. The operating system's Information Management Officer (IMO)/Information System
+Security Officer (ISSO) and System Administrators (SAs) must be notified via email and/or
+monitoring system trap when there is an unauthorized modification of a configuration item.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="aide_scan_notification"/>
+<ref prodtype="rhel7" nist="CM-3(5)" disa="1744"
+ossrg="SRG-OS-000363-GPOS-00150" sitgid="020140" />
+</Rule>
+
+<Rule id="aide_verify_acls" severity="medium" prodtype="rhel7">
+<title>Configure AIDE to Verify Access Control Lists (ACLs)</title>
+<description>
+By default, the <tt>acl</tt> option is added to the <tt>FIPSR</tt> ruleset in AIDE.
+If using a custom ruleset or the <tt>acl</tt> option is missing, add <tt>acl</tt>
+to the appropriate ruleset.
+For example, add <tt>acl</tt> to the following line in <tt>/etc/aide.conf</tt>:
+<pre>FIPSR = p+i+n+u+g+s+m+c+acl+selinux+xattrs+sha256</pre>
+AIDE rules can be configured in multiple ways; this is merely one example that is already
+configured by default.
+</description>
+<ocil clause="the acl option is missing or not added to the correct ruleset">
+To determine that AIDE is verifying ACLs, run the following command:
+<pre>$ grep acl /etc/aide.conf</pre>
+Verify that the <tt>acl</tt> option is added to the correct ruleset.
+</ocil>
+<rationale>
+ACLs can provide permissions beyond those permitted through the file mode and must be
+verified by the file integrity tools.
+</rationale>
+<ident prodtype="rhel7" cce="CCE-TBD" />
+<oval id="aide_verify_acls"/>
```

Comment:
Opened https://github.com/OpenSCAP/scap-security-guide/issues/1565 to track OVAL generation